websites/F/Font Awesome/dist/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Font Awesome",
+    "author": {
+      "name": "OGMatrix",
+      "id": "657290880947978241"
+    },
+    "category": "other",
+    "logo": "https://i.imgur.com/SnQ9HYz.png",
+    "thumbnail": "https://i.imgur.com/GJFmAgg.png",
+    "version": "1.0.0",
+    "color": "#0285f7",
+    "tags": [
+      "ogmatrix",
```

Comment:
``ogmatrix``, delete this tag. This tag is not affiliated with this website, or is it?